akka-stream/src/main/scala/akka/stream/Materializer.scala

```diff
@@ -145,6 +146,16 @@ abstract class Materializer {
 
 }
 
+object Materializer {
+
+  /**
+   * Implicitly provides the system wide materializer
+   */
+  implicit def matFromSystem(implicit system: ActorSystem): Materializer =
+    SystemMaterializer(system).materializer
```

Comment:
Can we do something for `typed.ActorSystem`?

Reply:
Still haven't figured anything out here.